crates/ide-completion/src/tests.rs

```diff
@@ -105,7 +105,7 @@ fn completion_list_with_config(
     include_keywords: bool,
     trigger_character: Option<char>,
 ) -> String {
-    // filter out all but one builtintype completion for smaller test outputs
+    // filter out all but one builtin type completion for smaller test outputs
```

Comment:
Nit (I know we use both):

```suggestion
    // filter out all but one built-in type completion for smaller test outputs
```

Reply:
Oddly, I really don't mind built-in. And I'm happy not to enforce it.

I can't remember if this change was me or Google Sheets...

Anyway, I'll apply and squash.